packages/design-system/src/components/N8nInfoTip/InfoTip.vue

```diff
@@ -23,39 +35,11 @@ const props = withDefaults(defineProps<InfoTipProps>(), {
 	tooltipPlacement: 'top',
 });
 
-const iconData = computed((): { icon: string; color: string } => {
-	switch (props.theme) {
-		case 'info':
-			return {
-				icon: 'info-circle',
-				color: '--color-text-light)',
-			};
-		case 'info-light':
-			return {
-				icon: 'info-circle',
-				color: 'var(--color-foreground-dark)',
-			};
-		case 'warning':
-			return {
-				icon: 'exclamation-triangle',
-				color: 'var(--color-warning)',
-			};
-		case 'danger':
-			return {
-				icon: 'exclamation-triangle',
-				color: 'var(--color-danger)',
-			};
-		case 'success':
-			return {
-				icon: 'check-circle',
-				color: 'var(--color-success)',
-			};
-		default:
-			return {
-				icon: 'info-circle',
-				color: '--color-text-light)',
-			};
-	}
+const iconData = computed<{ icon: IconMap[keyof IconMap]; color: IconColor }>(() => {
+	return {
+		icon: ICON_MAP[props.theme],
+		color: props.theme === 'info' || props.theme === 'info-light' ? 'text-base' : props.theme,
```

Comment:
I used `text-base` here as it looks the same for the specific icon I'm testing, but I'm not positive this is equivalent? Note the changed snapshot test - there we now have `text-base` rather than nothing due to the previous invalid value.

Will `text-base` always be correct, or do we risk "shadowing" an existing value in a parent somewhere?